### PR TITLE
Validate the address provided to --ticketaddress

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

**Comment by jcvernaleo** [comment]
This needs to be rebased.

File: config.go [hunk 1]
```diff
@@ -502,6 +502,17 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	if len(cfg.TicketAddress) != 0 {
+		_, err := dcrutil.DecodeAddress(cfg.TicketAddress, activeNet.Params)
+		if err != nil {
+			str := "%s: ticketaddress '%s' failed to decode: %v"
+			err := fmt.Errorf(str, funcName, cfg.TicketAddress, err)
+			fmt.Fprintln(os.Stderr, err)
```
Comment (jcvernaleo) [inline config.go]: Shouldn't you use the logger instead of a fmt.Print here?

**Comment by jcvernaleo** [comment]
Other than that those two things it works and seems like a good idea to me.

File: config.go (hunk 1 above)
Comment (dajohi) [inline config.go]: No, config.go happens before logger is setup.

**Comment by jcvernaleo** [comment]
Rebase looks good and I guess you are right about the logging.
Works fine so OK.

**Pull request merged**